mopidy_youtube/backend.py

```diff
@@ -49,7 +52,7 @@ def resolve_url(url, stream=False):
     try:
         video = pafy.new(url)
         if not stream:
-            uri = 'youtube:video/%s.%s' % (
```

Comment:
If I understand it correctly, the intention of using slash in the URI here was to help a number of clients to show just the last part of the URI as the name, e.g. "title foo bar baz.the_id_we_actually_use". Changing the `/` to `:` will break that.